js/screens/ActivityDetails/ActivityDetailsContainer.js

```diff
@@ -4,21 +4,30 @@ import ActivityDetails from "./ActivityDetails";
 export default class ActivityContainer extends Component {
   constructor(props) {
     super(props);
+    this.state = {
+      isVisible: false
+    };
   }
   static navigationOptions = {
     title: "Activity",
     headerStyle: {
-      backgroundColor: "#bbded6"
+      backgroundColor: "#bbded6",
+      borderBottomColor: "transparent"
     },
     titleStyle: {
       color: "#fff"
```

Comment:
Some places in the code use "white", some places use "#fff". Use one or the other. Also, move all colors into your theme css.